src/modules/AnalysisExecution/const.js

```diff
@@ -172,14 +172,20 @@ const submissionActionTypes = keyMirror({
 const importableAnalysisTypes = ['COHORT', 'ESTIMATION', 'PREDICTION'];
 const analysisTypeNames = {
   COHORT: 'cohort',
-  ESTIMATION: 'estimation',
-  PREDICTION: 'prediction',
+  ESTIMATION: 'PLE analysis',
+  PREDICTION: 'PLP analysis',
 };
-function nameAnalysisType(analysisType, capitalize = false) {
-  if (!(analysisType in analysisTypeNames)) {
+const pluralAnalysisTypeNames = {
+  COHORT: 'cohorts',
+  ESTIMATION: 'PLE analyses',
+  PREDICTION: 'PLP analyses',
+};
+function nameAnalysisType(analysisType, capitalize = false, plural = false) {
```

Comment:
I would accept object and use destructuring to make function calls more obvious (by usage of named params, because sequence of true / falses is not very transparent)